overlord/snapstate/snapstate.go

```diff
@@ -455,6 +456,47 @@ func CheckChangeConflict(st *state.State, snapName string, checkConflictPredicat
 	return nil
 }
 
+func contentAttr(attrs map[string]interface{}) string {
+	// we always have a "content" attr in plug/slots of interface
+	// type "content"
+	s, _ := attrs["content"].(string)
```

Comment:
Same comment re `plug|slot.Attr(..)` (if you need the helper anyway, the method can take interfaces.Attrer interface type.